data/mods/innawood/mining/furniture_and_terrain/veins.json

```diff
@@ -10,5 +10,214 @@
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_niter",
+    "name": "niter",
+    "description": "A vein of saltpeter.",
+    "symbol": "#",
+    "color": "light_gray",
+    "move_cost": 0,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "MINEABLE" ],
+    "roof": "t_rock_floor",
+    "bash": {
+      "str_min": 100,
+      "str_max": 400,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_rock_floor",
+      "items": [ { "item": "rock", "count": [ 3, 7 ] }, { "item": "material_niter", "count": [ 1, 3 ], "prob": 80 } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_brimstone",
+    "name": "brimstone",
+    "description": "A vein of sulfur.",
+    "symbol": "#",
+    "color": "yellow",
+    "move_cost": 0,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "MINEABLE" ],
+    "roof": "t_rock_floor",
+    "bash": {
+      "str_min": 100,
+      "str_max": 400,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_rock_floor",
+      "items": [ { "item": "rock", "count": [ 3, 7 ] }, { "item": "chunk_sulfur", "count": [ 1, 3 ], "prob": 80 } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_copper",
+    "name": "native copper",
+    "description": "A vein of copper.",
+    "symbol": "#",
+    "color": "light_red",
+    "move_cost": 0,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "MINEABLE" ],
+    "roof": "t_rock_floor",
+    "bash": {
+      "str_min": 100,
+      "str_max": 400,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_rock_floor",
+      "items": [ { "item": "rock", "count": [ 3, 7 ] }, { "item": "chunk_copper", "count": [ 1, 3 ], "prob": 80 } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_silver",
+    "name": "native silver",
+    "description": "A vein of silver.",
+    "symbol": "#",
+    "color": "white",
+    "move_cost": 0,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "MINEABLE" ],
+    "roof": "t_rock_floor",
+    "bash": {
+      "str_min": 100,
+      "str_max": 400,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_rock_floor",
+      "items": [ { "item": "rock", "count": [ 3, 7 ] }, { "item": "chunk_silver", "count": [ 1, 2 ], "prob": 80 } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_gold",
+    "name": "native gold",
+    "description": "A glittering vein of gold.",
+    "symbol": "#",
+    "color": "yellow",
+    "move_cost": 0,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "MINEABLE" ],
+    "roof": "t_rock_floor",
+    "bash": {
+      "str_min": 100,
+      "str_max": 400,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_rock_floor",
+      "items": [ { "item": "rock", "count": [ 3, 7 ] }, { "item": "chunk_gold", "count": [ 1, 2 ], "prob": 80 } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_rock_aluminum",
+    "name": "native alumimum",
+    "description": "A vein of alumimum.",
```

Comment:
Did you mean `aluminium` here?

Reply:
Yes, thank you.
Created #55657 to fix it.